code/_onclick/observer.dm

```diff
@@ -8,7 +8,7 @@
 			return									// seems legit.
 
 	// Things you might plausibly want to follow
-	if((ismob(A) && A != src) || istype(A,/obj/machinery/bot) || istype(A,/obj/singularity))
+	if((ismob(A) && A != src) || istype(A,/obj/machinery/bot) || istype(A,/obj/singularity) || istype(A,/obj/mecha))
```

Comment:
Let's just have both, normal follow and orbiting. Bonus if one day we can have humans see ghosts and have them all congregate into a single shell haunting